api/src/main/java/jakarta/data/Order.java

```diff
@@ -95,6 +95,15 @@ public static final <T> Order<T> by(Sort<T>... sorts) {
         return new Order<T>(List.of(sorts));
     }
 
+    /**
+     * The instances of {@link Sort} belonging to this {@code Order}.
+     *
+     * @return the instances of {@link Sort}
```

Comment:
Updating JavaDoc to let the user know the list of Sorts follows the order of precedence that they provided just in case they aren't comfortable assuming so.
```suggestion
     * @return the instances of {@link Sort}, from highest precedence to lowest precedence.
```